utils/addresses.ts

```diff
@@ -1,4 +1,4 @@
-interface IfcOneNetworkAddresses {
+interface InterfaceNetworkTokenAddresses {
```

Comment:
This is a convention we use elsewhere to prefix interfaces with `I` instead of `Interface` so for consistency may be best to rewrite this to:
```suggestion
interface INetworkTokenAddresses {
```